.github/workflows/drivers.yml

```
@@ -25,6 +25,17 @@ env:
   DRIVERS_BUCKET: ${{ inputs.drivers-bucket }}
 
 jobs:
+  # This sentinel job exists such that this workflow call "runs" even
+  # if subsequent driver build steps are skipped. This means that other
+  # workflows can depend on this workflow and still run even if it is skipped
+  #
+  # This only affects release branches, so is limited to those.
+  sentinel:
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref_name, 'release-')
```

This will only trigger on the `release-*` branch, but that doesn't tag the versioned collector which is the one we need for releases. You might want to replace this check or complement it with `github.ref_type == 'tag'`
```suggestion
    if: startsWith(github.ref_name, 'release-') || github.ref_type == 'tag'
```